JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,28 @@
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa.
+
+### Generate Similar Tabs Based on a Data Source       
+
+To generate similar tabs and views, bind the TabPanel to data. Use the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) or [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property to do this. Both these properties can work with local arrays, but **dataSource** also accepts a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. You can use this object if you need to process the local array or fetch the array from a remote data source. In this demo, the **dataSource** property is set to a local array.
+
+Each object in the **items[]** or **dataSource** array can contain predefined fields, such as **title** or **icon** (see the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) section for the full list). The TabPanel automatically recognizes these fields and creates the default tab and view appearance based on them. However, if your data objects contain custom fields, you need to specify the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) and [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate) that define how to display the fields in tabs and views. This demo illustrates the latter use case.
+
+### Create Individual Tabs
+
+If you want each tab and view to have differently structured content, define individual templates. To do this, assign an array of objects to the **items[]** or **dataSource** property and specify the [tabTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) and [template](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) properties in each object. This use case is illustrated in the following tutorial: [Getting Started with TabPanel]().
+
+### Switch Between Tabs
+
+To switch between tabs programmatically, use the [selectedIndex](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#selectedIndex) property. It accepts the index of a tab in the **dataSource** or **items[]** array. The [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#onSelectionChanged) function allows you perform desired actions when the selected tab changes.
+
+Use the following properties to configure user navigation between tabs:
+
+- [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled)      
+Defines whether to switch between views with a swipe gesture.
```

Comment:
```suggestion
Defines whether or not to switch between views with a swipe gesture.
```

Reply:
Hello. We have already discussed this phrase a few months ago (please see the screenshots or the [discussion in Teams](https://teams.microsoft.com/l/message/19:b8c5aa2758c44e2db10312fae66683f2@thread.skype/1617040695260?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=9298adba-77e3-4b3e-a0eb-b3c426b4ba6d&parentMessageId=1617005840357&teamName=DX%20Tech%20Writers%20Club&channelName=doc.guidelines&createdTime=1617040695260)). Has anything changed? 
![image](https://user-images.githubusercontent.com/33040934/122260334-d284a480-cedb-11eb-9495-9f423f9c2da4.png)
![image](https://user-images.githubusercontent.com/33040934/122260348-d6b0c200-cedb-11eb-966f-1f3578f3bf1e.png)

Reply:
Yes I know. But with the word " between " there, I think the sentence works better this way. but i don't mind if you don't keep the addition.